src/components/icons/clear-icon.tsx

```diff
@@ -0,0 +1,8 @@
+export const ClearIcon = (props: React.SVGProps<SVGSVGElement>) => (
```

Comment:
1. This is basically a smaller version of existing `CloseIcon`. Let's call it `CloseSmallIcon`.

Or rename both to `CrossIcon` / `CrossSmallIcon` to name them based on appearance rather than based on how they're used. Because otherwise we'll have a bunch of identical looking icons named Close, Delete, Clear, Dismiss, Error.

Reply:
renamed